lectures/networks.md

```diff
@@ -406,7 +406,7 @@ G_p.in_degree('p')
 
 ### Communication
 
-Next we study communication and connectedness, which have important
+Next, we study communication and connectedness, which have important
```

Comment:
@HengchengZhang , thanks for these changes.

I tend to bend the rule of grammar for simplicity, and so that the exposition flows smoothly.  So I don't put commas after "Next" and often I skip one after "In fact".  This is quite common in American English.

Reply:
Thanks for pointing this out, I will keep in mind in the following lectures.